tests/modules/annotations/resources/test_annotation_keywords.py

```diff
@@ -29,17 +26,18 @@ def test_keywords_on_annotation(
     with db.session.begin():
         db.session.add(keyword)
     assert keyword is not None
-
```

Comment:
The cloned Asset group has no sightings so is effectively an invaid asset group. So we really need to replace all uses of the cloned AGs with creating ones like below.